e2e/apprunner/copilot/back-end/manifest.yml

```diff
@@ -0,0 +1,57 @@
+# The manifest for the "back-end" service.
+# Read the full specification for the "Request-Driven Web Service" type at:
+# https://aws.github.io/copilot-cli/docs/manifest/rd-web-service/
+
+# Your service name will be used in naming your resources like log groups, App Runner services, etc.
+name: back-end
+# The "architecture" of the service you're running.
+type: Request-Driven Web Service
+
+image:
+  # Docker build arguments.
+  # For additional overrides: https://aws.github.io/copilot-cli/docs/manifest/rd-web-service/#image-build
+  build: back-end/Dockerfile
+  # Port exposed through your container to route traffic to it.
+  port: 80
+
+http:
+  private: true
+  # healthcheck:
+  #   path: /
+  #   healthy_threshold: 3
+  #   unhealthy_threshold: 5
+  #   interval: 10s
+  #   timeout: 5s
+
+# Number of CPU units for the task.
+cpu: 1024
+# Amount of memory in MiB used by the task.
+memory: 2048
+
+# Connect your App Runner service to your environment's VPC.
+network:
+  vpc:
+    placement: private
+
+
+# Enable tracing for the service.
+# observability:
+#   tracing: awsxray
+
+# Optional fields for more advanced use-cases.
+#
+# variables:                    # Pass environment variables as key value pairs.
+#   LOG_LEVEL: info
+#
+# tags:                         # Pass tags as key value pairs.
+#   project: project-name
+#
+secrets:                      # Pass secrets from AWS Systems Manager (SSM) Parameter Store and Secrets Manager.
+  USER_CREDS: 
+    secretsmanager: 'e2e-apprunner-MyTestSecret'  # The key is the name of the environment variable, the value is the name of an SSM parameter.
```

Comment:
Is there any reason why you wanted to add this secret to the backend manifest, instead of frontend manifest?

If not, can we:
1. delete `back-end/manifest.yml`
2.  move this to secret `front-end/manifest.yml` so that it has:
```
secrets:                      # Pass secrets from AWS Systems Manager (SSM) Parameter Store and Secrets Manager.
  my-ssm-param: e2e-apprunner-ssm-param
  USER_CREDS:
    secretsmanager: 'e2e-apprunner-MyTestSecret'
```


Unless necessary, I'd love us to maintain less manifests for e2e tests.

Reply:
@iamhopaul123 suggested I do that. I can add them both to just one manifest too.
Doest that work @iamhopaul123 ?

Reply:
Yeah i agreed with @Lou1415926 's idea. Let's do it

Reply:
DOne